policy/release/base_image_registries.rego

```diff
@@ -0,0 +1,70 @@
+#
+# METADATA
+# title: Base image checks
+# description: |-
+#   This package is responsible for verifying the base (parent) images
+#   reported in the attestation are acceptable.
+#
+package policy.release.base_image_registries
+
+import future.keywords.contains
+import future.keywords.if
+import future.keywords.in
+
+import data.lib
+import data.lib.bundles
+
+# METADATA
+# title: Restrict registry of base images
+# description: |-
+#   The base images used when building a container image must come from a known set
+#   of trusted registries to reduce potential supply chain attacks. This policy
+#   defines trusted registries as registries that are fully maintained by Red Hat
+#   and only contains content produced by Red Hat.
```

Comment:
grammar nitpick: "and only contain content..."

Reply:
```suggestion
#   and only contain content produced by Red Hat.
```

Reply:
Thanks!